src/gui/socketapi/socketapi.cpp

```diff
@@ -866,8 +882,7 @@ void SocketApi::sendSharingContextMenuOptions(const FileData &fileData, SocketLi
     // If sharing is globally disabled, do not show any sharing entries.
     // If there is no permission to share for this file, add a disabled entry saying so
     if (isOnTheServer && !record._remotePerm.isNull() && !record._remotePerm.hasPermission(RemotePermissions::CanReshare)) {
-        listener->sendMessage(QLatin1String("MENU_ITEM:DISABLED:d:") + (!record.isDirectory()
-            ? tr("Resharing this file is not allowed") : tr("Resharing this folder is not allowed")));
+        listener->sendMessage(QLatin1String("MENU_ITEM:DISABLED:d:") + (!record.isDirectory() ? tr("Resharing this file is not allowed") : tr("Resharing this folder is not allowed")));
```

Comment:
This looks like a "Verschlimmbesserung". 
maybe break it up into two real lines instead of a single statement.

Reply:
Clang format does that for some reason and as I moved the file it was touched...

Reply:
Well then move `(!record.isDirectory() ? tr("Resharing this file is not allowed") : tr("Resharing this folder is not allowed"))` to a variable?

Reply:
I didn't change that line I can also try to revert the format change....